Test/CouponTest.cs

```diff
@@ -159,6 +159,25 @@ public void Coupon_plan_must_exist()
             create.ShouldThrow<ValidationException>();
         }
 
+        [RecurlyFact(TestEnvironment.Type.Integration)]
+        public void CreateCouponItem()
+        {
+            var item = new Item(GetMockItemCode("coupon item"), "Coupon Item");
+            item.Create();
+
+            var anotherItem = new Item(GetMockItemCode("another item"), "Another Item");
+            anotherItem.Create();
+
+            var coupon = new Coupon(GetMockCouponCode(), GetMockCouponName(), new Dictionary<string, int>());
+            coupon.DiscountInCents.Add("USD", 200);
+            coupon.Items.Add(item.ItemCode);
+            coupon.Items.Add(anotherItem.ItemCode);
```

Comment:
Since it is being added, is it worth flexing `AppliesToAllItems` here? It seems like a basic value passthrough, so not sure how much mileage it would net.

Reply:
Good call. I think it's worth adding the coverage.